tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +118,196 @@ module('Acceptance | guid file | registration files', hooks => {
+        editDescriptionTextArea.textContent = 'A test description.';
+        assert.dom('[data-test-description-field]')
+            .containsText('A test description.', 'Metadata description field updates.');
+        // Update resource type
+        await selectChoose('[data-test-select-resource-type]', 'InteractiveResource');
+        // Update resource language
+        await selectChoose('[data-test-select-resource-language]', 'Abkhazian');
+        await selectChoose('[data-test-select-resource-language]', 'Arabic');
+        await selectChoose('[data-test-select-resource-language]', 'Chinese');
+        await selectChoose('[data-test-select-resource-language]', 'English');
+        await selectChoose('[data-test-select-resource-language]', 'French');
+        await selectChoose('[data-test-select-resource-language]', 'German');
+        await selectChoose('[data-test-select-resource-language]', 'Hindi');
+        await selectChoose('[data-test-select-resource-language]', 'Italian');
+        await selectChoose('[data-test-select-resource-language]', 'Japanese');
+        await selectChoose('[data-test-select-resource-language]', 'Korean');
+        await selectChoose('[data-test-select-resource-language]', 'Latin');
+        await selectChoose('[data-test-select-resource-language]', 'Multiple languages');
+        await selectChoose('[data-test-select-resource-language]', 'North American Indian languages');
+        await selectChoose('[data-test-select-resource-language]', 'Polish');
+        await selectChoose('[data-test-select-resource-language]', 'Portuguese');
+        await selectChoose('[data-test-select-resource-language]', 'Russian');
+        await selectChoose('[data-test-select-resource-language]', 'South American Indian languages');
+        await selectChoose('[data-test-select-resource-language]', 'Spanish');
+        await selectChoose('[data-test-select-resource-language]', 'Swedish');
+        await selectChoose('[data-test-select-resource-language]', 'Tagalog');
+        await selectChoose('[data-test-select-resource-language]', 'Thai');
+        await selectChoose('[data-test-select-resource-language]', 'Vietnamese');
+        await selectChoose('[data-test-select-resource-language]', 'Zuni');
+        // Screenshot after changes
+        await percySnapshot(assert);
+        // Save changes
+        await click('[data-test-save-metadata-button]');
+        // Verify form closes
+        assert.dom('[data-test-edit-metadata-form]').doesNotExist();
+        assert.dom('[data-test-metadata-tab]').exists();
+    });
+
+    // Verify anonymous registrations
+    test('anonymous registration', async function(this: ThisTestContext, assert) {
+        const node = server.create('node', {
+            isAnonymous: true,
+        });
+
+        await visit(`/--node/${node.id}`);
+        assert.equal(currentURL(), `/--node/${node.id}`);
+
+        assert.dom('[data-test-metadata-node]').doesNotExist();
+    });
```

Comment:
This test doesn't work. The method to create an anonymous node is slightly different from this. 

The reason the assertion passes is because you're going to a page that doesn't have the metadata on it and checking for the non-existence a data-test selector that is unique to the metadata section. Best to just remove this for now and save it for a future ticket.

Reply:
Ok, removing this to be included in a future ticket.